lib/opensusebasetest.pm

```diff
@@ -374,23 +375,20 @@ sub wait_boot {
             assert_screen "grub2", 15;
         }
         elsif (match_has_tag("migration-source-system-grub2") or match_has_tag('grub2')) {
-            send_key "ret";    # boot to source system
+            # no action required
```

Comment:
if no action required why you not just drop whole branch ? I would understand if you would have final else at the end but this is not the case

Reply:
You are right. I assumed L400 and the following to be an else-branch which I must not hit but actually it's "elsif (!match_has_tag('grub'))" so I should delete this elsif-branch